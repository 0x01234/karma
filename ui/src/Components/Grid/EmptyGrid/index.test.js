import React from "react";

import { shallow } from "enzyme";

import toDiffableHtml from "diffable-html";

import { EmptyGrid } from ".";

describe("<EmptyGrid />", () => {
  it("matches snapshot", () => {
    const tree = shallow(<EmptyGrid />);
    expect(toDiffableHtml(tree.html())).toMatchSnapshot();
  });
});
