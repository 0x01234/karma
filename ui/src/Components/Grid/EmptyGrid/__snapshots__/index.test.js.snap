// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyGrid /> matches snapshot 1`] = `
"
<h1 class=\\"display-1 text-secondary screen-center\\">
  <svg aria-hidden=\\"true\\"
       focusable=\\"false\\"
       data-prefix=\\"fas\\"
       data-icon=\\"check-circle\\"
       class=\\"svg-inline--fa fa-check-circle fa-w-16 fa-null fa-rotate-null fa-pull-null \\"
       role=\\"img\\"
       xmlns=\\"http://www.w3.org/2000/svg\\"
       viewbox=\\"0 0 512 512\\"
       style=\\"font-size:14rem\\"
  >
    <path fill=\\"currentColor\\"
          d=\\"M504 256c0 136.967-111.033 248-248 248S8 392.967 8 256 119.033 8 256 8s248 111.033 248 248zM227.314 387.314l184-184c6.248-6.248 6.248-16.379 0-22.627l-22.627-22.627c-6.248-6.249-16.379-6.249-22.628 0L216 308.118l-70.059-70.059c-6.248-6.248-16.379-6.248-22.628 0l-22.627 22.627c-6.248 6.248-6.248 16.379 0 22.627l104 104c6.249 6.249 16.379 6.249 22.628.001z\\"
    >
    </path>
  </svg>
</h1>
"
`;
